js/slv/quadrant.js

```diff
@@ -23,6 +23,15 @@ function Quadrant(id, row, col, map, lngDivisions, latDivisions) {
 }
 
 Quadrant.findQuadrant = function (lng, lat, incX, incY) {
+    // SC-733 : works only if lng is in [-180,180] or lng can put computed as 220 for instance on the map -> add check on lng
+    lng = lng % 360; // first ensure to be in [0:360]
```

Comment:
It seems that the modulo operator in javascript is not a real modulo (but the remainder) and may return a negative value (see [this page](https://stackoverflow.com/questions/4467539/javascript-modulo-gives-a-negative-result-for-negative-numbers)). So this line may have no effect at all (except ensure that `lng` is not greater that 360 nor lower than -360). That may be why you had to add the `else if` below to handle negative value. But I am not a javascript expert.